### PR TITLE
Sync option

### LINKED ISSUE #39: Per-computer enable/disable
I have one computer in a dark room and another in a bright room. It would be nice if there was an option of enabling Dark Reader on only one of them, even when they use the same Chrome account.

### LINKED ISSUE #444: Option for NOT syncing settings
I want to turn on the darkreader on my desktop computer, but turn off it on my laptop computer.
For more.
Further,  can we add some profiles and switch between the profiles in the different instances of chrome?

### PR DESCRIPTION
**Pull request by Gusted**
- Add option to disable/enable syncing of options
- Resolves #444
- Resolves #39

### CODE REVIEW

File: src/ui/popup/miscellaneous-settings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import {isFirefox} from '../../../utils/platform';
+import {ViewProps} from '../types';
+import EnabledByDefaultGroup from './enabled-by-default';
+import InvertPDF from './invert-pdf';
+import SyncSettings from './Sync-settings';
+
+export default function MiscellaneousSettingsPage(props: ViewProps) {
+    return (
+        <section class="m-section">
+            <EnabledByDefaultGroup {...props} />
```
Comment (alexanderby) [inline src/ui/popup/miscellaneous-settings/index.tsx:11]: This "Enabled by default" is one of the most important settings, I think it should be at the top of others. Maybe the "Site list" page will be on top and this option will be moved there.

File: src/ui/popup/settings-page/index.tsx [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
     onManageSettingsClick: () => void;
+    onMiscellaneousNavClick: () => void;
 };
 
 export default function SettingsPage(props: SettingsPageProps) {
     return (
         <section class="m-section">
-            <EnabledByDefaultGroup {...props} />
-            {isFirefox() ? null : <InvertPDF {...props} />}
+            <MiscellaneousButton onClick={props.onMiscellaneousNavClick}/>
```
Comment (alexanderby) [inline src/ui/popup/settings-page/index.tsx:17]: Maybe let's not add another page this time, but put the new option into "Manage Settings" page? Sounds like a suitable place for me.

File: src/ui/popup/miscellaneous-settings/Sync-settings.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function SyncSettings(props: ViewProps) {
+    function onSyncSettingsChange(checked: boolean) {
+        props.actions.changeSettings({syncSettings: checked});
+    }
+
+    return (
+        <CheckButton
+            checked={props.data.settings.syncSettings}
+            label="Enable syncing settings"
```
Comment (alexanderby) [inline src/ui/popup/miscellaneous-settings/Sync-settings.tsx:13]: Change to "Enable settings sync"

File: src/ui/popup/miscellaneous-settings/Sync-settings.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            description={props.data.settings.syncSettings ?
+                'Enabled to sync settings' :
```
Comment (alexanderby) [inline src/ui/popup/miscellaneous-settings/Sync-settings.tsx:15]: Change to "Synchronized across devices"/"Not synchronized across devices" (if it's not too long).

**Comment by alexanderby** [comment]
A funny thing is, that if user chooses `syncSettings: false`, this setting will never be written into sync storage and user will keep using sync storage. I suggest you to add some `console.log` to debug what happens.

**Comment by Gusted** [comment]
Their you go whe on change of the sync option it will force to save it to both ends.

**Pull request merged**